docs/vNext/CONTRIBUTING.md

```diff
@@ -211,6 +210,46 @@ Each adapter extends the `ConnectorBase` from the core package, as well as expor
 
 This is the integration test suite used for caliper; it runs in the Azure pipelines build and can (*should*) be run locally when checking code changes. Please see the readme within the package for more details.
 
+## Dependency management in the monorepo
+
+### npm version
+
+For developing Caliper, the minimum required version of npm is `7.24.2`. This is because npm 7 introduced the `npm workspaces` feature, which is used to manage the Caliper monorepo.
+
+In case the npm version is not sufficient, the following error may be thrown when trying to install the dependencies:
+
+```console
+user@ubuntu:~/caliper$ npm ci
+npm ERR! code ENOTSUP
+npm ERR! notsup Unsupported engine for caliper@0.5.1-unstable: wanted: {"node":">=14.19.0","npm":">=7.24.2"} (current: {"node":"14.19.3","npm":"6.14.17"})
+npm ERR! notsup Not compatible with your version of node/npm: caliper@0.5.1-unstable
+npm ERR! notsup Not compatible with your version of node/npm: caliper@0.5.1-unstable
+npm ERR! notsup Required: {"node":">=14.19.0","npm":">=7.24.2"}
+npm ERR! notsup Actual:   {"npm":"6.14.17","node":"14.19.3"}
+```
+
+If the npm version is lower than `7.24.2`, you can update it to latest by running the following command:
+
+```console
+npm install -g npm@latest
+```
+
+### Adding new dependencies
```

Comment:
Is it similar for updating and uninstalling unused depencencies ?

Reply:
Yes. I have now updated the documentation to include uninstall instructions.